### PR TITLE
Tronxy x5sa v5 v6

### PR DESCRIPTION
**Pull request by rhapsodyv**
Configs for Tronxy X5SA.

Works together with: https://github.com/MarlinFirmware/Marlin/pull/18243

### CODE REVIEW

File: config/examples/Tronxy/X5SA/HOWTO-INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+```
+M6046 ; sdcard access
+M8512 "currentconfig.gcode" ; save settings to file
+```
+2. Save it on the printer's SD card
+3. Put the card in the printer and "print" this file
+4. The printer won't do anything. Just wait a few seconds and stop the print.
+5. Your current printer settings are stored in the file: `currentconfig.gcode`
+
+You can read more about it in [this guide](https://www.facebook.com/notes/tronxy-turnigy-x5s-x5sa-x3s-3d-printer-drucker-users/tronxy-firmware-configuration-guide-by-keith-varin-addermk264bit-tuning/649799805579765/).
```
Comment (thinkyhead) [inline config/examples/Tronxy/X5SA/HOWTO-INSTALL.md:28]: This link is broken.

Comment (rhapsodyv) [reply config/examples/Tronxy/X5SA/HOWTO-INSTALL.md:28]: The link works for group members. I just put it in the tutorial to properly credit the author.

File: config/examples/Tronxy/X5SA/HOWTO-INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+```
+To:
+```cpp
+__weak void board_setup_gpio(void) {
+    GPIOA->regs->BSRR = (1U << PA14);
+    gpio_init_all();
+}
+```
+
+A patch may already have been applied to Marlin by the time you read this.
```
Comment (rhapsodyv) [inline config/examples/Tronxy/X5SA/HOWTO-INSTALL.md:256]: I already have a PR for this. But first some users are testing to see if really solves the problems for everyone.
In one or two days I send the PR.

**Comment by rhapsodyv** [comment]
The fix some user are testing: https://github.com/rhapsodyv/Marlin/commit/a43ecdd0030ecdb6ad3bc340b59fe8ddc745c768

File: config/examples/Tronxy/X5SA/HOWTO-INSTALL.md (hunk 1 above)
Comment (thinkyhead) [reply config/examples/Tronxy/X5SA/HOWTO-INSTALL.md:28]: Could we have a link to something that is "open source"?

**Comment by thinkyhead** [comment]
The fix looks sensible, and the comment is helpful. Has a PR been posted yet? I'm just getting started here….

File: config/examples/Tronxy/X5SA/HOWTO-INSTALL.md (hunk 1 above)
Comment (rhapsodyv) [reply config/examples/Tronxy/X5SA/HOWTO-INSTALL.md:28]: Anyone can join the group (https://www.facebook.com/groups/169728600253557/about)

Comment (rhapsodyv) [reply config/examples/Tronxy/X5SA/HOWTO-INSTALL.md:28]: The main propose here is just credit the author. This "save" methods was discovered by this user, and posted there.

**Review by rhapsodyv — COMMENTED** [review]
> The fix looks sensible, and the comment is helpful. Has a PR been posted yet? I'm just getting started here….

The fix works. Just waiting a few more users with different boards to test, to see if we need more hacks or if we can get rid of the `#if CHITU_V5_Z_MIN_BUGFIX` .

**Pull request merged**